### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by zach-betz-hln** [privatenumber/pkgroll#56] [General]
Just wanted to drop by and say thank you for this **sweet**, sweet library 🙏 

Our small company has a few internal packages, and now I'm able to write them in TS, yet publish them as CJS, MJS, and their types 🚀

**Comment by privatenumber** [comment]
Thanks for sharing your kind words! 🙏

I think it can help a lot more people—I'd really appreciate it if you can help spread the word!

**Reply by zach-betz-hln** [reply]
Would be happy to 💪
